packages/lit-dev-content/samples/examples/async-task/npm-info.ts

```diff
@@ -0,0 +1,109 @@
+import {LitElement, css, html} from 'lit';
+import {customElement, state} from 'lit/decorators.js';
+import {Task} from '@lit-labs/task';
+
+@customElement('npm-info')
+export class NpmInfo extends LitElement {
+  @state()
+  _package = 'lit';
+
+  _fetchNpmInfoTask = new Task(this, {
+    task: async ([pkgName], {signal}) => {
+      if (pkgName === undefined || pkgName === '') {
+        throw new Error('Empty package name');
+      }
+      // Artifical delay for demo purposes
+      await new Promise((r) => setTimeout(r, 1000));
+      const response = await fetch(getPackageUrl(pkgName), {signal});
+      if (response.status === 200) {
+        return response.json() as Promise<NpmPackage>;
+      } else {
+        throw response.text();
+      }
+    },
+    args: () => [this._package],
+  });
+
+  _onChange(e: Event) {
```

Comment:
I'd mvoe this to below the render method

Reply:
done